Riot/Modules/Room/DataSources/RoomDataSource.m

```diff
@@ -973,15 +972,20 @@ - (void)resetAccessibilityForCell:(MXKRoomBubbleTableViewCell *)cell
     cell.attachmentView.accessibilityLabel = nil;
 }
 
-#pragma mark - Threads
+#pragma mark - MXThreadingServiceDelegate
 
-- (void)newThreadCreated:(NSNotification *)notification
+- (void)threadingService:(MXThreadingService *)service didCreateNewThread:(MXThread *)thread direction:(MXTimelineDirection)direction
 {
     if (self.threadId)
     {
         //  no need to reload the thread screen
         return;
     }
+    if (direction == MXTimelineDirectionBackwards)
```

Comment:
As commented [here](https://github.com/matrix-org/matrix-ios-sdk/pull/1381#discussion_r814103492) I don't fully understand the relationship between `direction` and the creation of a thread. Does it mean something like where in the timeline the thread should be placed?

Reply:
I've tried to explain it on your comment.